code/messaging/src/main/java/com/adobe/marketing/mobile/aepcomposeui/AepUIConstants.kt

```diff
@@ -26,4 +26,22 @@ internal object AepUIConstants {
             const val SPACING = 8
         }
     }
+
+    internal object Keys {
```

Comment:
Can we move these to MessagingConstants? These are not used by UI library right?